hdfs-commons/src/main/java/org/apache/mesos/hdfs/file/FileUtils.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.mesos.hdfs.file;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+
+/**
+ * Used for file system operations
+ */
+public final class FileUtils {
+
+  private static final Logger LOG = LoggerFactory.getLogger(FileUtils.class);
+
+  private FileUtils() {
+  }
+
+  public static void createDir(File dataDir) {
+    if (dataDir.exists()) {
+      LOG.warn("data dir exits:" + dataDir);
```

Comment:
In this case it should be a log.info because we call this method expecting that the dataDir might already be there. We might rename this to createDirIfEmpty or some other more descriptive method